components/2023/home/sections/contact.js

```diff
@@ -33,9 +33,9 @@ export default function Contact() {
             <i className="bi bi-geo-alt"></i>
             <h3 className="text-lg uppercase font-bold">Address</h3>
             <address>
-              NWSC International Resource Center, Bugolobi (IREC)
-              <br /> Plot M11 Old Portbell Road, Bugolobi, <br />
-              P. O. Box 7053, Kampala, Uganda
+            International University Of East Africa
+              {/* <br /> Plot M11 Old Portbell Road, Bugolobi, <br />
```

Comment:
thanks @Leon12-aaryson for the updates. what do you think about completely deleting this versus commenting given the change of venue wont be reversible